node/rules/typescript.md

```diff
@@ -0,0 +1,8 @@
+
+## Configuration Selected
+We have extended the default [Ratehub](https://github.com/ratehub/code-style/tree/master/typescript/rules) rule set. Deviations from the rule set will be discussed below.
+
+## Rule Deviations
+* [[max-len]](https://eslint.org/docs/rules/max-len) | ERROR | Limit line length to 120 characters, except for strings, URLs, and regular expressions.
```

Comment:
This is just re-stating the rule: the spirit of the method here is to explain *why* it’s setup like it is. I would assume it’s something to do with attempting to keep standard lines at easily readable lengths for most monitors?